Sources/MessagingPush/PushHandling/iOSPushEventListener.swift

```diff
@@ -37,7 +27,7 @@ protocol PushEventListener: AutoMockable {
  - During the native iOS SDK's initialization, the SDK's digraph instance is re-created. All objects (and singletons) in that old digraph instance are deleted from memory.
  - That's bad! If the PushEventListener singleton instance was stored in the digraph, it would be deleted from memory. The OS would no longer be able to send push notification callbacks to the SDK.
  */
-class iOSPushEventListener: PushEventListener {
+class iOSPushEventListener: PushEventHandler {
```

Comment:
The goal with all of this de-coupling is that *all* push handling logic in our SDK is inside of this 1 file. All automated tests can be executed against this file to test all the use cases.